woocommerce/woocommerce-gateway-stripe
# Restrict Link payment method to US merchants

Fixes #2445

## Changes proposed in this Pull Request:
Payments using Link payment method should be available only for US merchants. This PR restricts the usage of Link payment method only to US merchants






## Testing instructions
* add a Stripe US regular account
* enable Link payment method
* test if the order process works as expected using Link payment method
* change the Stripe account with a non-US account
* Link payment method should not be visible in admin 
* Link payment method should not be active on checkout page




---


**Post merge**
